### PR TITLE
Fixing the previous bug in redirection

### PR DESCRIPTION
**Pull request by Elena-alexeenko**

### CODE REVIEW

**Review by soryy708 — CHANGES_REQUESTED** [review]
This doesn't work when you're in the `/login` route

File: src/front/app/routes/login.js [hunk 1]
```diff
@@ -37,7 +37,19 @@ class LoginRoute extends React.Component {
...
-                    this.props.history.push('/');
+                    //console.log(this.props.history.location.pathname);
+                    if(this.props.history.location.pathname==='/')
+                    {
+                        this.props.history.go('/#/');
+                    }
+                    if(this.props.history.location.pathname==='/login')
+                    {
+                        this.props.history.push('/');
+                        window.location.reload();
```
Comment (soryy708) [inline src/front/app/routes/login.js:47]: Why is a `.reload` here neccesary?

Comment (Elena-alexeenko) [reply src/front/app/routes/login.js:47]: the page is stuck without the reload

Comment (soryy708) [reply src/front/app/routes/login.js:47]: I just tested this, and I see what you mean.

**Review by soryy708 — APPROVED** [review]

**Pull request merged**